packages/codegen/src/__tests__/testCodeGen.uint.spec.ts

```diff
@@ -0,0 +1,174 @@
+// import path from 'path';
```

Comment:
Having `test` in the name is a bit redundant as it's `.spec.` that matters. So I suggest renaming these files to just `codeGen.uint.spec.ts` etc)